BaragonCore/src/main/java/com/hubspot/baragon/models/BaragonService.java

```diff
@@ -22,7 +22,7 @@
   private final Collection<String> owners;
 
   @NotNull
-  @Pattern(regexp = "/(?:[A-Za-z0-9\\-._~!$&'()*+,;=:@/?]|%[0-9A-Fa-f]{2})*", message = "must be an absolute URL path")
+  @Pattern(regexp = "[a-zA-Z_0-9\\-]+(\\.\\w[a-zA-Z_0-9\\-]+)+(/[#&\\n\\-=?\\+\\%/\\.\\w]+)?|/(?:[A-Za-z0-9\\-._~!$&'()*+,;=:@/?]|%[0-9A-Fa-f]{2})*", message = "must be an absolute URL path")
```

Comment:
Might make sense, I did it this way initially so we leverage the same locking on base path we already had without adding a second field that had to be checked to determine if the path was locked.